web/src/engine/osk/src/input/gestures/specsForLayout.ts

```diff
@@ -632,7 +632,8 @@ export function specialKeyEndModel(params: GestureParams): GestureModel<any> {
 export function longpressModel(params: GestureParams, allowShortcut: boolean, allowRoaming: boolean): GestureModel<any> {
   const base: GestureModel<any> = {
     id: 'longpress',
-    resolutionPriority: 0,
+    // Needs to beat flick-start priority.
+    resolutionPriority: 4,
```

Comment:
For reference, `flick-start` is specified at priority 3.

https://github.com/keymanapp/keyman/blob/b4ed832ed93710b91b1791179e0d9efaf24d9805/web/src/engine/osk/src/input/gestures/specsForLayout.ts#L731-L734